benchexec/tools/template.py

```diff
@@ -151,6 +151,24 @@ def determine_result(self, returncode, returnsignal, output, isTimeout):
         """
         return result.RESULT_UNKNOWN
 
+    def determine_result_for_property(self, returncode, returnsignal, output, isTimeout, property):
+        """
+        Parse the output of the tool and extract the verification result for specified property.
+        This method always needs to be overridden for multi-property verification.
+        If the tool gave a result, this method needs to return one of the
+        benchexec.result.RESULT_* strings.
+        Otherwise an arbitrary string can be returned that will be shown to the user
+        and should give some indication of the failure reason
+        (e.g., "CRASH", "OUT_OF_MEMORY", etc.).
+        @param returncode: the exit code of the program, 0 if the program was killed
+        @param returnsignal: the signal that killed the program, 0 if program exited itself
+        @param output: a list of strings of output lines of the tool (both stdout and stderr)
+        @param isTimeout: whether the result is a timeout
+        (useful to distinguish between program killed because of error and timeout)
+        @param property: property, for which status is expected
+        @return a non-empty string, usually one of the benchexec.result.RESULT_* constants
```

Comment:
I think the most important thing is to have it consistently. So I would not like `unreach-call(__VERIFIER_error_linux_usb_urb): false` in combination with `valid-deref: false(valid-deref)`.

On the other hand, `valid-deref: false(valid-deref)` seems unnecessarily redundant anyway. So I would prefer `valid-deref: false` for the yaml file.

For the result XML (and the HTML tables) I am not sure. You might have experience looking at such tables: Is repeating the property name in cases like `valid-deref: false(valid-deref)` helpful, or does it clutter the table too much?

For the return value of `determine_result_for_property`, we could either simply use `false` as well, or we use `false(valid-deref)`. The latter would make it behave the same as `determine_result`, which I like, because in the future we could merge the two methods maybe?